### ISSUE TITLE
Can i use PostMan to send a notification?

### ISSUE THREAD

**Issue by mustufa84** [Ankitagupta2309/pushKnot#3]
Instead of using the protractor script in the example Notify.js, can i use postman tool to send the notification?

If yes then where should i pass the Authorization information. When i passed in via headers, it did not work.

**Comment by Ankitagupta2309** [comment]
Yes, you can use postman to send notifications. The authorization is usually assed via Headers only, but if its not working, can you check with the notification team as to how they are passing the authorization?
